plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoMetadata.java

```diff
@@ -652,7 +805,7 @@ private ConnectorTableMetadata getTableMetadata(SchemaTableName tableName)
     private static List<MongoColumnHandle> buildColumnHandles(ConnectorTableMetadata tableMetadata)
     {
         return tableMetadata.getColumns().stream()
-                .map(m -> new MongoColumnHandle(m.getName(), m.getType(), m.isHidden(), Optional.ofNullable(m.getComment())))
+                .map(m -> new MongoColumnHandle(m.getName(), ImmutableList.of(), m.getType(), m.isHidden(), false, Optional.ofNullable(m.getComment())))
```

Comment:
Why are we hard-coding `false` for `dbRef` argument here? `m.getType()` might be `dbref` type.

Reply:
`buildColumnHandles` gets used during table creation. So, identifying the dbRef column is not required. That is why I had hard-corded `false` here.

I will modify the code to set `dbRef` according to `m.getType()`.

Reply:
Changed meaning of `dbRefField`. It is the field inside the DBRef column. It's not required in `buildColumnHandles`.

```
    // Represent if the field is inside a DBRef type column
    private final boolean dbRefField;
```